doc/releases/nautilus.rst

```diff
@@ -1,3 +1,361 @@
+v14.2.5 Nautilus
+================
+
+This is the fifth release of the Ceph Nautilus release series. All Nautilus
+users are advised to upgrade to this release.
+
+Notable Changes
+---------------
+
```

Comment:
This release fixes a critical BlueStore bug (link) introduced in 14.2.3 (and also present in 14.2.4) that can lead to data corruption when a separate 'wal' device is used.

Reply:
@ifed01 what else should we say here?

Reply:
@liewegas - IMO this is good enough.

Reply:
> This release fixes a critical BlueStore bug (link) introduced in 14.2.3 (and also present in 14.2.4) that can lead to data corruption when a separate 'wal' device is used.

https://tracker.ceph.com/issues/42223